delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/indexing/QueryGraph.scala

```diff
@@ -43,7 +34,7 @@ final class QueryGraph(client: BlazegraphClient, namespace: String, query: Sparq
         Some(g.replaceRootNode(id))
       }
 
-  override def apply(element: SuccessElem[GraphResource]): Task[Elem[GraphResource]] =
+  def apply(element: Elem[GraphResource]): Task[Elem[GraphResource]] =
```

Comment:
We can get rid of the elem here as it is not a pipe anymore

Reply:
Removed